WhaleSpotting/react-app/src/tests/Footer.test.tsx

```diff
@@ -0,0 +1,21 @@
+import { render, screen } from "@testing-library/react";
+import { Footer } from "../components/Footer";
+import React from "react";
+
+test("renders Whale Museum logo in footer", () => {
+    render(<Footer />);
+
+    const whaleMuseumLogo = screen.getByTestId("whale-museum-logo");
+
+    expect(whaleMuseumLogo).toBeInTheDocument();
+    expect(whaleMuseumLogo.src === "https://cdn.shopify.com/s/files/1/0249/1083/t/3/assets/wm-logo-sm.png").toBeTruthy();
```

Comment:
We will get better error messages if we do something like the below, and in next test as well 
```suggestion
    expect(whaleMuseumLogo.src).toBe("https://cdn.shopify.com/s/files/1/0249/1083/t/3/assets/wm-logo-sm.png");
```